lib/module.js

```diff
@@ -19,6 +19,10 @@ async function redirectModule (moduleOptions) {
 }
 
 async function parseOptions (options) {
```

Comment:
```suggestion
async function parseOptions (options = {}) {
```

If setting a default value resolves the issue too, I'd prefer that way :relaxed:

Reply:
The code will then return an empty array that will be spread into object but that will result in no-op, same as return empty object. So I'm fine with that too.